coral-schema/src/main/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverter.java

```diff
@@ -53,7 +53,7 @@ public static ViewToAvroSchemaConverter create(HiveMetastoreClient hiveMetastore
   }
 
   /**
-   * This is the main API to generate the avro schema for a given Dali view
+   * An API to generate the avro schema for a view
```

Comment:
Hive table or view? cc: @wmoustafa

Reply:
I am okay either way.

Reply:
Searching the entire code base for `tableOrViewName`, only this file pops up. My preference is to stick to view in this file as well, and change the variable name too.